docs/source/en/tutorials/autopipeline.md

```diff
@@ -34,13 +47,17 @@ image = pipeline(prompt, num_inference_steps=25).images[0]
 
 Under the hood, [`AutoPipelineForText2Image`]:
 
-1. automatically detects a `"stable-diffusion"` class from the [`model_index.json`](https://huggingface.co/runwayml/stable-diffusion-v1-5/blob/main/model_index.json) file
-2. loads the corresponding text-to-image [`StableDiffusionPipline`] based on the `"stable-diffusion"` class name
+1. Automatically detects a `"stable-diffusion"` class from the [`model_index.json`](https://huggingface.co/runwayml/stable-diffusion-v1-5/blob/main/model_index.json) file.
+2. Loads the corresponding text-to-image [`StableDiffusionPipeline`] based on the `"stable-diffusion"` class name.
```

Comment:
No need to change anything here

Reply:
Reversed except for a typo at StableDiffusionPipline.